### PR TITLE
Fix enriched contacts

### PR DESCRIPTION
**Pull request by vivekbhr**
* [x] Flake8 passes (`flake8 . --exclude=.venv,.build,planemo_test_env,build --ignore=E501,F403,E402,F999,F405,E712`)
* [x] Local tests pass (`py.test hicexplorer --doctest-modules`)

added `-m ` to `findEnrichedcontacts`
close pool, after multiproc in hicFindTADs (helps with memory)
renamed the option `--depth` to `--maxDepth` in  `findEnrichedcontacts` for the sake of consistency everywhere (galaxy needs fix?)

### CODE REVIEW

File: hicexplorer/hicFindEnrichedContacts.py [hunk 1]
```diff
@@ -812,8 +815,8 @@ def main(args=None):
         hic_ma.diagflat()
 
     if args.method == 'obs/exp':
-        hic_ma.convert_to_obs_exp_matrix(maxdepth=args.depth, perchr=args.perchr)
+        hic_ma.convert_to_obs_exp_matrix(maxdepth=args.maxDepth, perchr=args.perchr)
     else:
-        hic_ma.convert_to_zscore_matrix(maxdepth=args.depth, perchr=args.perchr)
+        hic_ma.convert_to_zscore_matrix(maxdepth=args.maxDepth, perchr=args.perchr)
```
Comment (joachimwolff) [inline hicexplorer/hicFindEnrichedContacts.py]: If the parameter is changed from `maxDepth` to `depth` than please change it in the source code too.

Comment (vivekbhr) [reply hicexplorer/hicFindEnrichedContacts.py]: it's the other way round.. it was called `--depth` in the argparse but maxDepth everywhere else. so I changed it to maxDepth..

**Comment by vivekbhr** [comment]
failed tests seem to be already failing in develop.. @joachimwolff shall we do something about fixing the tests?

File: hicexplorer/hicFindEnrichedContacts.py (hunk 1 above)
Comment (joachimwolff) [reply hicexplorer/hicFindEnrichedContacts.py]: You are right, my fault.

**Comment by joachimwolff** [comment]
We have to at some point but not now, it is imho one of these fails introduced by small changes in the plotting libs.

**Pull request merged**